pkgs/top-level/all-packages.nix

```diff
@@ -13805,6 +13803,9 @@ with pkgs;
   scala_2_11 = callPackage ../development/compilers/scala/2.x.nix { majorVersion = "2.11"; jre = jdk8; };
   scala_2_12 = callPackage ../development/compilers/scala/2.x.nix { majorVersion = "2.12"; };
   scala_2_13 = callPackage ../development/compilers/scala/2.x.nix { majorVersion = "2.13"; };
+  scala_3 = callPackage ../development/compilers/scala { };
+  # deprecated
+  dotty = scala_3;
```

Comment:
This should have been added to `pkgs/top-level/aliases.nix`, see https://github.com/NixOS/nixos-search/issues/680

Reply:
good point, done in #250383